src/Microsoft.AspNet.Razor/Generator/GeneratedClassContext.cs

```diff
@@ -124,8 +141,10 @@ public struct GeneratedClassContext
         // Required Items
         public string WriteMethodName { get; private set; }
         public string WriteLiteralMethodName { get; private set; }
+        public string WriteTagHelperMethodName { get; private set; }
         public string WriteToMethodName { get; private set; }
         public string WriteLiteralToMethodName { get; private set; }
+        public string WriteTagHelperToMethodName { get; private set; }
         public string ExecuteMethodName { get; private set; }
         public GeneratedTagHelperContext GeneratedTagHelperContext { get; private set; }
```

Comment:
nits:
- unless these are serialized somewhere along the line (@NTaylorMullen ?), can remove the private setters
- might also be worth adding blank lines between these properties, giving them some room to breath / be read